src/components/Events.js

```diff
@@ -16,12 +16,30 @@ const mockEvents = [
     name: "Honza",
     sport: "Tennis",
   },
+  {
+    id: 4,
+    name: "Pepa",
+    sport: "Rugby",
+  },
+  {
+    id: 5,
+    name: "Fanda",
+    sport: "Traktor",
+  },
+  {
+    id: 6,
+    name: "Jarmila",
+    sport: "Golf",
+  },
 ];
 
+const renderedEvents = mockEvents.map((mockEvent) => (
+  <li key={mockEvent.id} className={style.event}>
+    <div style={{ fontWeight: "bold" }}>{mockEvent.sport}</div>
+    <div>From: {mockEvent.name}</div>
+  </li>
+));
+
 export const Events = () => {
-  return (
-    <div className={style.container}>
-      <div>Events</div>
-    </div>
-  );
+  return <ul className={style.list}>{renderedEvents}</ul>;
```

Comment:
Doing this separation makes sense, when the `renderedEvents` are inside the render, that means 

```
export const Events = () => {
 // here <-----
  return (
```

it would get created on every render, so it would reflect any changes made to it. The problem right now, it will never get recreated, which might cause issues.

Generally, you don't need this separation unless you are repeating the same events multiple times, in that moment it should be a component probably. 

So it is fine just to inline it on line 44

Reply:
understood, fixed